website/events-listener/tsconfig.json

```diff
@@ -3,9 +3,9 @@
     "esModuleInterop": true,
     "moduleResolution": "node",
     "strict": true,
-    "target": "ESNext",
+    "target": "esNext",
```

Comment:
I'm not against it, but genuinely — why? 🤔

Reply:
I did this accidentally. Just was trying different options